doc/mediasdk-man.md

```diff
@@ -9146,6 +9334,8 @@ SDK API 1.27 adds `MFX_PLATFORM_ICELAKE`.
 
 SDK API 1.31 adds `MFX_PLATFORM_ELKHARTLAKE`, `MFX_PLATFORM_JASPERLAKE`, `MFX_PLATFORM_TIGERLAKE`.
 
+SDK API 1.34 adds `MFX_PLATFORM_KEEMBAY`.
```

Comment:
I don't see `MFX_PLATFORM_KEEMBAY` in the API headers .h files! Did you forget to add?

Reply:
Good point, yes, need to add.

Reply:
done